.github/workflows/pika.yml

```
@@ -84,59 +84,44 @@ jobs:
-
-      - name: Set up Go
-        uses: actions/setup-go@v5
-        with:
-          go-version: 1.19
-
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          node-version: '14'
```

actionlint

input "node-version" is not defined in action "actions/checkout@v3". available inputs are "clean", "fetch-depth", "fetch-tags", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "submodules", "token"

(action)